src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -147,8 +145,9 @@ public Future<Budget> getBudgetByFundIdAndFiscalYearId(String fiscalYearId, Stri
           handleFailure(promise, reply);
         } else if (reply.result()
           .isEmpty()) {
-          logger.error(BUDGET_NOT_FOUND_FOR_TRANSACTION);
-          promise.fail(new HttpException(Response.Status.BAD_REQUEST.getStatusCode(), BUDGET_NOT_FOUND_FOR_TRANSACTION));
+          logger.error(BUDGET_NOT_FOUND_FOR_TRANSACTION.toError());
```

Comment:
Can you add to logs fiscal year id and fund Id, you can use in logs your custom message

Reply:
Ok, I will fixt